Userland/Libraries/LibWeb/WebGL/WebGLRenderingContextBase.h

```diff
@@ -9,14 +10,16 @@
 #include <AK/RefCountForwarder.h>
 #include <AK/WeakPtr.h>
 #include <AK/Weakable.h>
-#include <LibGL/GLContext.h>
 #include <LibJS/Heap/GCPtr.h>
 #include <LibWeb/Bindings/PlatformObject.h>
 #include <LibWeb/Forward.h>
+#include <LibWeb/WebGL/OpenGLContext.h>
 #include <LibWeb/WebGL/WebGLContextAttributes.h>
 
 namespace Web::WebGL {
 
+#define GL_NO_ERROR 0
```

Comment:
Should this really be defined here? :thonk:

Reply:
as I see we will have to define WebGL constants from spec and use them instead.
for now I defined `GL_NO_ERROR` here so OpenGL header does not have to be included in this file. spec also guarantees this constant to be zero so it's probably ok.

Reply:
But nothing in this file uses `GL_NO_ERROR`?

Reply:
it is used here https://github.com/SerenityOS/serenity/blob/bfa4e68191d0e9be531b43bf3e6109ad42aba07b/Userland/Libraries/LibWeb/WebGL/WebGLRenderingContextBase.h#L97

before this change it was coming from LibGL's header that is not included in this file anymore.

Reply:
Oh jeez fine all right :)